test/ios_coverage_test.sh

```diff
@@ -166,6 +166,14 @@ function test_standalone_unit_test_coverage() {
   assert_contains "SharedLogic.m:-\[SharedLogic doSomething\]" "test-testlogs/app/standalone_test/coverage.dat"
 }
 
+function test_standalone_unit_test_coverage_json() {
+  create_common_files
+  do_coverage ios --test_output=errors --ios_minimum_os=9.0 --experimental_use_llvm_covmap --test_env=COVERAGE_PRODUCE_JSON=1 //app:standalone_test || fail "Should build"
+  cd "test-testlogs/app/standalone_test/test.outputs" || fail "Should unzip"
+  unzip "outputs.zip"
+  assert_contains "\"name\":\"SharedLogic.m:-\[SharedLogic doSomething\]\"" "coverage.json"
```

Comment:
```suggestion
  unzip_single_file "test-testlogs/app/standalone_test/test.outputs/outputs.zip" coverage.json \
      grep '"name":"SharedLogic.m:-\[SharedLogic doSomething\]"'
```

I think this is the same? similar to below? if the grep fails it should fail the test. not sure how much of a difference it makes but it should be more ephemeral than the current version